alembic/command.py

```diff
@@ -298,7 +298,8 @@ def retrieve_migrations(rev, context):
 
     if diffs:
         raise util.AutogenerateDiffsDetected(
-            f"New upgrade operations detected: {diffs}"
+            f"New upgrade operations detected: {diffs}",
+            revision_context=revision_context,
```

Comment:
would it make sense to pass also the diffs here? i know you can re-obtain that, but it's really easy to add that too

Reply:
I could but I thought keeping it light would be better, I'm gonna add it